apps/site/lib/site_web/templates/customer_support/_call_us.html.eex

```diff
@@ -0,0 +1,21 @@
+<p>
+  Monday &mdash; Friday: 6:30 AM &mdash; 8 PM<br>
+  Saturday &mdash; Sunday: 8 AM &mdash; 4 PM
+</p>
+
+<p>
+  <strong>Main Hotline:</strong> <%= tel_link "617-222-3200" %><br>
+  <strong>Toll Free:</strong> <%= tel_link "800-392-6100" %><br>
+  <strong>TTY:</strong> <%= tel_link "617-222-5146" %>
+</p>
+
+<p>
+  <strong>Elevator/Escalator Hotline:</strong> <%= tel_link "617-222-2828" %><br>
+</p>
+
+<p>
+  <strong>Lost and Found:</strong>
```

Comment:
It seems like (in the mockups attached on the ticket) Lost and Found might have been updated to be its own section, and also that additional headers have been added to this content (Hours, Support)... these changes don't appear to be assigned to other tickets so you should probably include them there.

Reply:
You're right, I didn't notice that. Will be creating a separate template for Lost and Found.